src/identity/interaction/email-password/handler/EmailPasswordResetPasswordRenderHandler.ts

```diff
@@ -0,0 +1,14 @@
+import { RenderHandler } from '../../../../server/util/RenderHandler';
+
+export interface EmailPasswordResetPasswordRenderHandlerProps {
+  errorMessage: string;
+  recordId: string;
+}
+
+/**
+ * A special render handler for the Reset Password form that
+ * includes the required props for rendering the reset password
+ * form.
+ */
+export abstract class EmailPasswordResetPasswordRenderHandler
```

Comment:
Do we need classes like these?

Reply:
Yes, there's quite a bit of information that needs to be extracted before you render the Reset Password page. If that information is incorrect, it should error instead of returning the page.